backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -295,6 +297,11 @@ public static Optional<Extension> convertToTribalAffiliationExtension(String tri
   public static Practitioner convertToPractitioner(Provider provider) {
     var practitioner = new Practitioner();
     practitioner.setId(provider.getInternalId().toString());
+    practitioner
+        .addIdentifier()
+        .setUse(IdentifierUse.OFFICIAL)
+        .setValue(provider.getProviderId())
```

Comment:
I think we have a decent amount of providers that do not have NPIs. since value is supposed to be unique, in that case would it be better to just not add an identifier to the practitioner or is RS requiring it?

Reply:
I noticed that value in a generated HL7 message from ReportStream and thought since we had the value as a "required" field we should add it too, but you are absolutely right that they don't have NPIs. I just assumed since it was required they did. 
<img width="268" alt="image" src="https://user-images.githubusercontent.com/10108172/219372143-25b06003-2f75-4727-abd5-72877d71c89a.png">